src/pages/MyBundlePage/MyBundle.style.ts

```diff
@@ -0,0 +1,42 @@
+import { css, styled } from 'styled-components';
+
+import { MOBILE } from '@/constants';
+
+export const Container = styled.article`
+  display: flex;
+  gap: 2rem;
+  padding: 4rem;
+  padding-top: 2rem;
+  width: 100%;
+  height: 100%;
+  box-sizing: border-box;
+  overflow-x: hidden;
+
+  @media screen and (max-width: ${MOBILE}px) {
+    height: auto;
+  }
+`;
+export const StyledWrapper = styled.section<{ $isSelected: boolean }>`
+  padding-top: 1rem;
+  box-sizing: border-box;
+  width: 20rem;
+  height: 100%;
+  display: flex;
+  flex-direction: column;
+  gap: 2rem;
+  transition:
+    transform 1s ease-in-out,
+    opacity 1s ease-in-out,
+    width 1s ease-in-out;
```

Comment:
```suggestion
    transform 0.5s ease-in-out,
    opacity 0.5s ease-in-out,
    width 0.2s ease-in-out;
```
P4; 트랜지션이 상위 태그, 하단 컴포넌트 태그 부분 모두 걸려있어서 약간 위쪽 태그 트랜지션을 빠르게 조절해도 좋을 것 같습니다.